subprojects/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -289,6 +295,52 @@ Gradle will treat these directories as possible installations but will not desce
 org.gradle.java.installations.paths=/custom/path/jdk1.8,/shared/jre11
 ----
 
+[NOTE]
+====
+Custom toolchains are not prioritized over <<sec:auto_detection,auto-detected>> ones. If auto-detection is turned on, custom toolchains
+can still be specified but all they will do is to extend the set of detected JRE/JDK locations. Among these locations, one will be picked
+according to the <<sec:precedence,Toolchain Precedence Rules>>.
+====
+
+[[sec:precedence]]
+== Precedence
+
+When there are multiple local JRE/JDK installations matching the toolchain specification of a build, one will be picked according to the
```

Comment:
I think this sentence should be clarified. Gradle will sort the toolchain in a stable order then check them through the predicate. The first that matches is then selected.